test/functionals/api/v2/containers_controller_test.rb

```diff
@@ -59,9 +61,49 @@ class ContainersControllerTest < ActionController::TestCase
         end
 
         test 'creates a container with correct params' do
+          repository_name = "centos"
+          tag = "7"
+          name = "foo"
+          registry_uri = URI.parse(@registry.url)
           Service::Containers.any_instance.expects(:pull_image).returns(true)
-          Service::Containers.any_instance.expects(:start_container).returns(true)
-          post :create, :container => { :name => 'foo', :registry_id => 3, :image => 'centos:7' }
+          Service::Containers.any_instance.expects(:start_container)
+                                          .returns(true).with do |container|
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 32) spaces for indenting an expression spanning multiple lines.